core/dynamic-tags/base-tag.php

```diff
@@ -171,7 +171,13 @@ protected function register_advanced_section() {}
 			'label' => __( 'Settings', 'elementor' ),
 		] );
 
-		$this->register_controls();
+		if ( $this->has_own_method( '_register_controls', self::class ) ) {
```

Comment:
It should be covered from the `Controls_Stack` itself because the same problem can occur on other objects that extend the `Controls_Stack`.

Reply:
There is no way to handle it from the `Controls_Stack`. Other occurrences have already been handled.